jb-yoda/src/test/java/life/jukebox/JBWatcherTest.java

```diff
@@ -0,0 +1,31 @@
+import static org.junit.Assert.*;
+import org.junit.*;
+import java.util.concurrent.*;
+import java.util.*;
+import java.io.*;
+
+public class JBWatcherTest {
+
+  private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+  JBWatcher watcher;
+  
+  @Before 
+  public void createWather() {
+    watcher = new JBWatcher("test-JBId");
+    System.setOut(new PrintStream(outContent));
+  }
+
+  @Test
+  public void testRun() {
+    watcher.run();
+    assertNotNull(outContent.toString());
```

Comment:
In general, be as specific with your tests as possible. Don't check for existence, check for validity of the message received.
Make the message watcher prints out a public static string. Then here check that you're receiving that specific string.

Reply:
Addressed in private msg. Will not be fixing.